docs/learn-balance-transfers.md

```diff
@@ -5,26 +5,51 @@ sidebar_label: How to transfer Balances
 ---
 
 Balance transfers are used to send one balance from one account to another account. To start
```

Comment:
One thing I noticed - since these are new images, could you delete the old ones from the balance transfers page?

Reply:
I'll take care of this in the big images PR, I have a script that will check for all orphan image files.